### PR TITLE
fix: VC Schema update to add uri constraint to issuer.id

### LINKED ISSUE #1737: Support only URI in VC Issuer.ID Schema
Currently, framework validates URI value when issuer is a string and doesn't validate the URI value when issuer is passed as issuer.id.

https://github.com/hyperledger/aries-framework-go/blob/3f763578a45b96f828f89857139f05acaf31b2ee/pkg/doc/verifiable/credential.go#L104-L121

### PR DESCRIPTION
**Pull request by rolsonquadras**
closes #1737 

Signed-off-by: Rolson Quadras <rolson.quadras@securekey.com>

### CODE REVIEW

File: pkg/doc/verifiable/credential_test.go [hunk 1]
```diff
@@ -355,6 +355,34 @@ func TestValidateVerCredIssuer(t *testing.T) {
 		require.Error(t, err)
 		require.Contains(t, err.Error(), "issuer: Invalid type")
 	})
+
+	t.Run("test verifiable credential with object context", func(t *testing.T) {
```
Comment (kdimak) [inline pkg/doc/verifiable/credential_test.go:359]: probably "with string issuer"

File: pkg/doc/verifiable/credential_test.go [hunk 2]
```diff
@@ -355,6 +355,34 @@ func TestValidateVerCredIssuer(t *testing.T) {
...
+		require.NoError(t, json.Unmarshal([]byte(validCredential), &raw))
+		raw.Issuer = "not-a-uri-issuer"
+		bytes, err := json.Marshal(raw)
+		require.NoError(t, err)
+		err = validateCredentialUsingJSONSchema(bytes, nil, &credentialOpts{})
+		require.Error(t, err)
+		require.Contains(t, err.Error(), "issuer: Does not match format 'uri'")
+	})
+
+	t.Run("test verifiable credential with object context", func(t *testing.T) {
```
Comment (kdimak) [inline pkg/doc/verifiable/credential_test.go:371]: probably "with object issuer"

File: pkg/doc/verifiable/credential_test.go (hunk 1 above)
Comment (rolsonquadras) [reply pkg/doc/verifiable/credential_test.go]: Missed to update the name. Thanks. I have updated it [now](https://github.com/hyperledger/aries-framework-go/compare/915c66c0ad4904b420459fef1797603726e67474..e7c610cb5ba36606a28670ce46fb69dc36bdcc10).

File: pkg/doc/verifiable/credential_test.go (hunk 2 above)
Comment (rolsonquadras) [reply pkg/doc/verifiable/credential_test.go]: [Done](https://github.com/hyperledger/aries-framework-go/compare/915c66c0ad4904b420459fef1797603726e67474..e7c610cb5ba36606a28670ce46fb69dc36bdcc10).

**Pull request merged**